### ISSUE TITLE
Accessibility/Alerts: Making alerts read automatically

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak-ui#3176]
### Description

The alerts (success, error etc.) are for a screen reader at the bottom of the page and would be useful if read automatically, e. g. had some `aria-live` attribute or something along these lines.

### Motivation

This enhancement was reported by a tester testing Admin UI accessibility in v 19.0.0.

**Comment by edewit** [comment]
But they already have this:
![image](https://user-images.githubusercontent.com/51133/186335349-78f6c677-839b-4eef-aab2-c0cb820d08c6.png)

**Comment by ssilvert** [comment]
Yes, this is an issue on Windows.  The NVDA reader does not read the alerts.

**Comment by ssilvert** [comment]
@edewit Are you pointing to PatternFly code above?  Is this a PatternFly issue?

**Comment by edewit** [comment]
yes this is for sure a patternfly issue we can't change how these components are rendered. So @ssilvert is the window tooling somehow not taking all these aria properties into account

**Comment by ssilvert** [comment]
@edewit Yes.  That's what I'm seeing and NOT hearing on Windows.  I'll open an issue with PatternFly and close this one.
